cmake/SundialsBuildOptionsPre.cmake

```diff
@@ -317,6 +317,11 @@ sundials_option(SUNDIALS_TEST_DEVTESTS BOOL
 sundials_option(SUNDIALS_TEST_UNITTESTS BOOL
   "Include unit tests in make test" OFF ADVANCED)
 
+# Include unit tests in regression tests
+sundials_option(SUNDIALS_TEST_DISABLE_GTEST BOOL
+  "Diable GTest unit tests" OFF ADVANCED)
```

Comment:
I am not sure that its clearer ... `DISABLE_GTEST` seems pretty clear as to what it is going to do. I used DISABLE because I like that it implies that you are disabling something that is normally enabled, which should encourage leaving GTest enabled unless it is necessary to disable.

Reply:
See my other comment [here](https://github.com/LLNL/sundials/pull/373#discussion_r1418691897)